.gitignore

```diff
@@ -11,3 +11,4 @@
 /externals/3rdparty/winpcap/Include
 /externals/3rdparty/winpcap/Lib
 
+CMakeFiles
```

Comment:
In which case would you have CMakeFiles inside your source tree?

Reply:
Just built on Mac with Makefile backend

Reply:
Hum, I just tried and don't have any CMakeFiles in my source folder.
Are you sure you are not generating your build folder inside the main source folder, which is absolutely not recommended?

This command doesn't cause any CMakeFiles to be seen by git
`cmake -H. -B_buildMakefiles -G"Unix Makefiles"`

Reply:
@christophe-calmejane  need to cleanup my dir on Mac. I was trying to build with different approaches and looks like got some mess inside the lib folder